pkg/commands/handler.go

```diff
@@ -36,6 +36,7 @@ func Handler() http.Handler {
 		r.Post("/readonly/enable", handleEnableReadonly)
 		r.Post("/readonly/disable", handleDisableReadonly)
 		r.Get("/dbuid", handleStolonDBUid)
+		r.Post("/haproxy/bounce", handleBounceHaproxy)
```

Comment:
I would maybe rename this to just `restart`. :)

Reply:
alas :)